### DISCUSSION TITLE
addDoubletScores error：CreateDimReducObject' is not an exported object from 'namespace:Seurat'

### DISCUSSION THREAD

**Discussion by tao-bioinfo** [GreenleafLab/ArchR#920] [Questions / Documentation]
```
> ArrowFiles
[1] "27_03.arrow" "27_05.arrow" "27_10.arrow" "30_03.arrow" "30_05.arrow"
[6] "30_10.arrow"

> doubScores <- addDoubletScores(input = ArrowFiles, k = 10, knnMethod = "UMAP", LSIMethod = 1)
```

Error：
```
<simpleError: 'CreateDimReducObject' is not an exported object from 'namespace:Seurat'>
```

[ArchR-addDoubletScores-856c42c03a5e-Date-2021-07-22_Time-12-11-52.log](https://github.com/GreenleafLab/ArchR/files/6859951/ArchR-addDoubletScores-856c42c03a5e-Date-2021-07-22_Time-12-11-52.log)

**Comment by rcorces** [comment]
In the future, please post bugs/errors in Issues and questions/feature requests in Discussions.

Maybe this is a Seurat version problem? `CreateDimReducObject` used to be exported by Seurat but now it is exported by `SeuratObject`. Maybe try down grading to something before Seurat v4? If this is the problem, I'm surprised others havent encountered this issue.

**Reply by rcorces** [reply]
I've converted this to an issue. Please follow up there https://github.com/GreenleafLab/ArchR/issues/921
